documentation/book/con-kafka-listeners.adoc

```diff
@@ -123,4 +123,44 @@ listeners:
 ----
 
 
-Authentication must be configured when using the User Operator to manage `KafkaUsers`.
\ No newline at end of file
+Authentication must be configured when using the User Operator to manage `KafkaUsers`.
+
+== Network Policies
+
+For every enabled listener, {ProductName} will automatically create a NetworkPolicy which will by default grant all applications access to the listener.
+You can restrict the access only to selected applications using the `networkPolicyPeers` field.
+You can use different `networkPolicyPeers` configuration for each listener.
+
+.An example where the `plain` and `tls` listeners allow connections only from selected pods
+[source,yaml,subs="attributes+"]
+----
+# ...
+listeners:
+      plain:
+        authentication:
+          type: scram-sha-512
+        networkPolicyPeers:
+          - podSelector:
+              matchLabels:
+                app: kafka-sasl-consumer
+          - podSelector:
+              matchLabels:
+                app: kafka-sasl-producer
+      tls:
+        authentication:
+          type: tls
+        networkPolicyPeers:
+          - namespaceSelector:
+              matchLabels:
+                project: myproject
+          - namespaceSelector:
+              matchLabels:
+                project: myproject2
+# ...
+----
+
+In the example above, only applications with matching labels will be allowed to connect to the `plain` listener and only applications from the namespaces matching the labels will be allowed to connect.
```

Comment:
I would explain better the above example distinguishing between the "plain" and the "tls". Something like ...

Only application pods matching the labels  "app: kafka-sasl-consumer" and "app: kafka-sasl-producer" can connect to the "plain" listener while only the application pods running in the namespaces matching the labels "project: myproject" and "project: myproject2" can connect to the "tls" listener.

Reply:
I changed the rather long sentence into bullet points ... do you mind checking it again?